django_filters/fields.py

```diff
@@ -210,7 +219,7 @@ def clean(self, value):
         return value
 
 
-class ChoiceIterator:
+class ChoiceIterator(BaseChoiceIterator):
```

Comment:
In Django 5.0 I added `django.utils.choices.ChoiceIterator` as a base class for the following:

- `django.forms.models.ModelChoiceIterator`
- `django.utils.choices.CallableChoiceIterator` (or `django.forms.fields.CallableChoiceIterator` in Django < 5.0)

It is expected that a subclass of this will handle normalization via use of `django.utils.choices.normalize_choices()` or manually (as is the case for `ModelChoiceIterator` which yields 2-tuples).